README.md

```diff
@@ -50,7 +50,7 @@ hatch run test
 In a typical ML-based timeseries analysis, the first step is always data processing.  A calendar-based datetime module `time` is implemented for time operations. For instance, a user is looking for predictors for winter climate at seasonal timescales (~180 days). First, a `calendar` object is created using `AdventCalendar`:
 
 ```py
->>> calendar = lilio.time.AdventCalendar(anchor="11-30", freq='180d')
+>>> calendar = lilio.AdventCalendar(anchor="11-30", freq='180d')
```

Comment:
Example outdated, AdventCalendar no longer exists.
There are several remaining references to the advent and other calendars in the README and docstrings.